src/components/comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getAllComments } from "../api";
+import { useParams } from "react-router-dom";
+
+const Comments = () => {
+  const [comments, setComments] = useState([]);
+  const { article_id } = useParams();
+
```

Comment:
would be nice to have a state to track whether the list of comments is loading and display a message accordingly (not obvious if there's an error or we are just waiting for comments to load on button click)